From 32488b31761a788f5ebf5677394c03256e945e89 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Tue, 12 Jul 2022 11:49:34 +0200
Subject: [PATCH 0589/1695] plat-stm32mp2: conf: enable VREFBUF

Enable VREFBUF compilation for STM32MP2

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 3089f394cfbafc0391e55fe3656715d73d2c3c6a plat-stm32mp2: conf: enable VREFBUF]
Change-Id: Ibd6504dcaf0d600f0fe9d1b86a86df16ab42401d
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/341945
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/359175
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 37ebdf7c4..c8fe52edc 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -106,6 +106,7 @@ CFG_STM32_SERC ?= y
 CFG_STM32_STGEN ?= y
 CFG_STM32_TAMP ?= y
 CFG_STM32_UART ?= y
+CFG_STM32_VREFBUF ?= y
 CFG_STPMIC2 ?= y
 
 CFG_DRIVERS_I2C ?= $(CFG_STM32_I2C)
-- 
2.47.2

